samples/multi-agent/src/main/java/demo/multiagent/application/AgentTeamWorkflow.java

```diff
@@ -7,7 +7,6 @@
 import static java.time.Duration.ofSeconds;
 import static java.time.temporal.ChronoUnit.SECONDS;
 
-// tag::all[]
```

Comment:
Another documentation issue silently introduced in #862, but making the build to fail now.